Tags  creating tag reader

ktran111 opened the pull request.

lawhvictoria committed:
Merge pull request #3 from Flashhhhhhhhhh/endpoints

Endpoints

ktran111 committed:
Merge branch 'tags--creating-tag-reader' of https://github.com/Flashhhhhhhhhh/backend into tags--creating-tag-reader

ktran111 committed:
server.py--fixed api endpoint for initial post

ktran111 committed:
api--fixed-broken-api-for-upload-csv

ktran111 committed:
Merge branch 'tags--creating-tag-reader' of https://github.com/Flashhhhhhhhhh/backend into tags--creating-tag-reader

tvillarete commented on src/CsvInterpreter.py:
```diff
@@ -1,59 +1,63 @@
-        examples_list += 'null],"tag":[null]}'
+        for i in range(min(len(self.examples), 10)):
+            examples_list += '"%s":[],' %(self.examples[i])
+        examples_list += '"tag":[null],"id":"%s"}' %(self.id)
```
We need to move `"id":"%s"` into `"tag":[]`, that way we don't have recursion issues.

Can we change this to:
```python
examples_list += '"tag": [{ "id": "%s" }]' %(self.id)
```

Commit:
CsvInterpreter: Move "id" into "tag" array

This also moves all of the tags into a JSON object, that way we can easily add
and remove tags without having to iterate through an array (key value instead)
